doc/application/application.rst

```diff
@@ -562,6 +591,10 @@ Make sure to follow these steps in order.
 
    More details are available below in :ref:`application_configuration`.
 
+#. If your application uses a Device Tree overlay file or files other than
+   the usual :file:`<board>.overlay`, add lines setting the
+   :makevar:`DTC_OVERLAY_FILE` variable to these files appropriately.
```

Comment:
Just to be explicit, how about:

```
set the CMake variable :makevar:`DTC_OVERLAY_FILE` to these files' paths,
with each path separated by whitespace (i.e., do not use a semicolon-separated
CMake list). Paths can be either absolute or relative to your application's source
directory.
```

(I'm assuming you want this to behave the same way CONF_FILE does -- though see below)